src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -143,11 +145,20 @@ export class CalciteSlider {
     const max = this.maxValue || this.value;
     const maxProp = this.isRange ? "maxValue" : "value";
     const value = this[maxProp];
+    const valueForLabeledHandle = value && value.toLocaleString();
+    const minValueForLabeledHandle = this.minValue && this.minValue.toLocaleString();
     const minInterval = this.getUnitInterval(min) * 100;
     const maxInterval = this.getUnitInterval(max) * 100;
     const mirror = this.shouldMirror();
-    const leftThumbOffset = `${mirror ? 100 - minInterval : minInterval}%`;
     const rightThumbOffset = `${mirror ? maxInterval : 100 - maxInterval}%`;
+    this.minValueZeroFlag = !!(this.isRange && !this.hasHistogram && this.minValue === 0);
```

Comment:
☝️ This looks like state is being modified in the render method.  Does stencil give you a warning when doing this, since it is not encouraged?

Reply:
@eriklharper I don't see any warning nope but I'll move this out, good call